app/uk/gov/hmrc/tradergoodsprofiles/controllers/RemoveRecordController.scala

```diff
@@ -58,18 +58,19 @@ class RemoveRecordController @Inject() (
   )(implicit ec: ExecutionContext): EitherT[Future, Result, RemoveRecordRequest] =
     EitherT.fromEither(
       request.body.validate[RemoveRecordRequest].asEither.leftMap { errors =>
-        InvalidErrorResponse(uuidService.uuid, InvalidRequestParameter, InvalidActorMessage, InvalidActorId).toResult
+        InvalidErrorsResponse(
+          uuidService.uuid,
+          Some(Seq(Error(InvalidRequestParameter, InvalidActorMessage, InvalidActorId)))
+        ).toResult
       }
     )
 
   def validateRecordId(recordId: String): EitherT[Future, Result, String] =
     EitherT.fromEither[Future](
       Try(UUID.fromString(recordId).toString).toEither.left.map(_ =>
-        InvalidErrorResponse(
+        InvalidErrorsResponse(
```

Comment:
It seem this is alway a 400 error. Can we rename maybe to BadRequestErrorResponse and refactor the object to inclide Code = BAD_REQUEST. Ping me if you want a chat about this